salt/modules/ipset.py

```diff
@@ -439,22 +444,31 @@ def check(set=None, entry=None, family='ipv4'):
                                                              ipaddress.ip_address(end))
                 entries = []
                 for network in networks:
+                    entries.append(network.with_prefixlen)
```

Comment:
I took that line out, I didn't mean to put it in. It was just a remnant from testing.